Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/Curvatures/interpolated_corrected_curvature_measures.h

```diff
@@ -0,0 +1,310 @@
+#ifndef CGAL_POLYGON_MESH_PROCESSING_INTERPOLATED_CORRECTED_CURVATURE_MEASURES_H
```

Comment:
You are missing a license header. Adapt the one from `CGAL/Polygon_mesh_processing/measure.h` for example.